packages/analytics-js-integrations/__tests__/integrations/Amplitude/browser.test.js

```diff
@@ -4,6 +4,7 @@ import Amplitude from '../../../src/integrations/Amplitude/browser';
 
 const amplitudeConfig = {
   apiKey: 'abcde',
+  proxyServerUrl : 'http://localhost:3000';
```

Comment:
this is http://, expected?

Reply:
this is just a test case. No, it is not expected. We are using console.error for the user to notify them use of insecured proxy URL.
<img width="1440" alt="Screenshot 2024-01-19 at 1 01 54 PM" src="https://github.com/rudderlabs/rudder-sdk-js/assets/60211312/675de5da-62a7-472d-9b49-e66622344366">

Reply:
While the UI configuration is :
<img width="1440" alt="Screenshot 2024-01-19 at 1 04 19 PM" src="https://github.com/rudderlabs/rudder-sdk-js/assets/60211312/492360df-04c1-4361-9883-37b8df7c61ab">

Reply:
edited